src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1036,13 +1036,10 @@ public async Task MigrateDatabaseOffline()
             }
 
             var dataDir = configuration.DataDirectory;
-            if (Directory.Exists(dataDir) == false)
```

Comment:
This method should also throw for non-Windows server. UI in Studio should be blocked